From 9c62d94f1e474922318b50f8db522c116236c542 Mon Sep 17 00:00:00 2001
From: Ido Ben-Hur <idoybh2@gmail.com>
Date: Fri, 6 Sep 2024 15:32:41 +0300
Subject: [PATCH] Revert "init.rc: remove 4.9/4.14 support"

This reverts commit a95be37d2cd3a126483ccd96e4162df8ff9ee1af.

Change-Id: I0a2d02c27cdb90d79a5d1f5035350ac94df3e49f
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 rootdir/init.rc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 70db67421fe7..92dcdd0131a4 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -682,6 +682,10 @@ on late-fs
     class_start early_hal
 
 # Only enable the bootreceiver tracing instance for kernels 5.10 and above.
+on late-fs && property:ro.kernel.version=4.9
+    setprop bootreceiver.enable 0
+on late-fs && property:ro.kernel.version=4.14
+    setprop bootreceiver.enable 0
 on late-fs && property:ro.kernel.version=4.19
     setprop bootreceiver.enable 0
 on late-fs && property:ro.kernel.version=5.4
